### PR TITLE
refactor: GlobalFooter and GlobalHeader

### PR DESCRIPTION
**Pull request by aionate0812**
### Overview

Updated header and footer components with the new url for our blog 

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [Reach-1017](https://codecademy.atlassian.net/browse/REACH-1017)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.9-alpha.42495b.0<br/>@codecademy/gamut-labs@19.0.2-alpha.42495b.0<br/>@codecademy/styleguide@41.1.2-alpha.42495b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60be2d4dd4b3f6aa1749c426--gamut-preview.netlify.app>https://60be2d4dd4b3f6aa1749c426--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60be2d4dd4b3f6aa1749c426>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.1.0-alpha.972931.0<br/>@codecademy/gamut-kit@0.4.9-alpha.972931.0<br/>@codecademy/gamut-labs@19.0.2-alpha.972931.0<br/>@codecademy/gamut@34.0.2-alpha.972931.0<br/>@codecademy/styleguide@41.2.0-alpha.972931.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60be300ec58849af78dff49f--gamut-preview.netlify.app>https://60be300ec58849af78dff49f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60be300ec58849af78dff49f>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterNavLinks/CompanyLinks.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ export const CompanyLinks: React.FC<CompanyLinksProps> = ({
             onClick={(event) => onClick({ event, target: 'jobs' })}
             variant="interface"
           >
-            We&apos;re Hiring
+            Careers
```
Comment (rogervera) [inline packages/gamut-labs/src/GlobalFooter/FooterNavLinks/CompanyLinks.tsx:94]: Is there a companion PR for this to be done in the monolith?

**Review by rogervera — APPROVED** [review]
Looks good!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.4.11-alpha.af59ce.0<br/>@codecademy/gamut-labs@19.0.4-alpha.af59ce.0<br/>@codecademy/styleguide@41.2.2-alpha.af59ce.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60be3745c6098487b68496a3--gamut-preview.netlify.app>https://60be3745c6098487b68496a3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60be3745c6098487b68496a3>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalFooter/FooterNavLinks/CompanyLinks.tsx (hunk 1 above)
Comment (aionate0812) [reply packages/gamut-labs/src/GlobalFooter/FooterNavLinks/CompanyLinks.tsx:94]: it's going to be a quick follow up :)

**Pull request merged**